### ISSUE TITLE
Enforce consistent use of aliases

### ISSUE THREAD

**Issue by brc-dd** [globalbrain/eslint-config#28]
Currently in our codebase we have all sort of import patterns like:

```ts
import foo from '../composables/Foo' // change to 👇
import foo from '@/composables/Foo'  // keep this
import foo from './Foo'              // maybe keep this too ? (same directory import)
```

Should we add a rule to make this consistent? There should be plugins available, or I can write one if we can reach a consensus.

**Comment by kiaking** [comment]
Yeah, general rule is that we use `./` for importing modules that are in the same scope like importing another component from a component.

I think `../` rarely happens, but I'm sure they are somewhere does that 👀 

Maybe most solid option would be to always enforce `@/` ? Sefirot can be optional I guess since it's a package 🤔

**Comment by kiaking** [comment]
Closing due to inactivity. I'm still fine with having lint rule on this but not having much issue either at the moment 😳
